src/components/Contacts/Contacts.tsx

```diff
@@ -1,7 +1,230 @@
 import React from 'react';
+import { ENDPOINTS } from '../../helpers/api';
+import { saveInLocalStorage } from '../../helpers/localStorage';
+import { emailRegEx, phoneRegEx } from '../../helpers/regex';
+import { Theme, ThemeContext } from '../../store/theme-context';
+import Card from './../UI/Card/Card';
+import Title from './../UI/Title/Title';
+import styles from './Contacts.module.scss';
+import Input from './Input/Input';
+import RadioInput from './RadioInput/RadioInput';
 
-const Contacts: React.FunctionComponent = () => {
-  return <div>Contacts</div>;
-};
+interface ContactsProps {}
+interface ContactsState extends InputValues {
+  formIsValid: boolean;
+  formIsSubmitted: boolean;
+  nameTouched: boolean;
+  emailTouched: boolean;
+  messageTouched: boolean;
+}
+interface InputValues {
+  [prop: string]: string | boolean | undefined;
+}
+interface Feedback {
+  id?: number;
+  name: string;
+  email: string;
+  message: string;
+  subscribe: boolean;
+  number?: string;
+}
+
+class Contacts extends React.Component<ContactsProps, ContactsState> {
+  thankYouCard: React.RefObject<HTMLDivElement>;
+  nameIsValid: boolean = false;
+  emailIsValid: boolean = false;
+  messageIsValid: boolean = false;
+  numberIsValid: boolean = false;
+  constructor(props: ContactsProps) {
+    super(props);
+    this.state = {
```

Comment:
missed?
https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/class_components/
```suggestion
    this.state: ContactsState = {
```

Reply:
IDE doesn't allow to add it here. Only in 32nd line.
![screenshot023](https://user-images.githubusercontent.com/96768501/179478634-df52aa15-8257-4835-949b-b59ae7b46432.png)

Reply:
seems like i shared an incorrect link :) it's fine, just leave your initial code version